<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleMaintainPage</class>
 <widget class="QWidget" name="VehicleMaintainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 50);</string>
  </property>
  <widget class="QLabel" name="LBLTC2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>40</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>H06</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>H03</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN8">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DataManage">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>数据
管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_HistoryFault">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>历史
故障</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN5">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>H04</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(19,20,87), stop:1 rgb(187,188,247))</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_ResetCut">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>切除
复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_MetroSet">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>315</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>车辆
设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN3">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN7">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN9">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTC1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>H01</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_RIOMTest">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>315</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>RIOM
调试</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBLMP1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>H02</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>305</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(187,188,247) , stop:1 rgb(19,20,87))</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN4">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLMP2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>140</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border:1px solid black;
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>H05</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNHome">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>620</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/Home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_TimeSet">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>时间
设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CommunicationStatus">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>通信
状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_SoftwareVersion">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>315</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>软件
版本</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DataMonitor">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>315</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>数据
监控</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_TractSubsystem">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>315</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>牵引
子系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: qlineargradient(x1:0, y1:0, x2:0, y2:1,stop:0 rgb(19,20,87), stop:1 rgb(187,188,247))</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_BrakeSubsystem">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>制动
子系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_AuxiliarySubsystem">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>辅助
子系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_BatterySubsystem">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>420</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>蓄电池
子系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_BogieLampSubsystem">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
color: rgb(255, 255, 255);
font:20px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>走行部
子系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_HMIInitSet">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>130</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 50);
border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lbl1</zorder>
  <zorder>lbl2</zorder>
  <zorder>lbl1_2</zorder>
  <zorder>LBLTC2</zorder>
  <zorder>LBLM1</zorder>
  <zorder>BTN8</zorder>
  <zorder>BTN_DataManage</zorder>
  <zorder>BTN_HistoryFault</zorder>
  <zorder>BTN5</zorder>
  <zorder>LBLM2</zorder>
  <zorder>BTN_ResetCut</zorder>
  <zorder>BTN1</zorder>
  <zorder>BTN_MetroSet</zorder>
  <zorder>BTN3</zorder>
  <zorder>BTN7</zorder>
  <zorder>BTN9</zorder>
  <zorder>LBLTC1</zorder>
  <zorder>BTN_RIOMTest</zorder>
  <zorder>LBLMP1</zorder>
  <zorder>BTN4</zorder>
  <zorder>LBLMP2</zorder>
  <zorder>BTN6</zorder>
  <zorder>BTNHome</zorder>
  <zorder>BTN2</zorder>
  <zorder>BTN_TimeSet</zorder>
  <zorder>BTN_CommunicationStatus</zorder>
  <zorder>BTN_SoftwareVersion</zorder>
  <zorder>BTN_DataMonitor</zorder>
  <zorder>BTN_TractSubsystem</zorder>
  <zorder>BTN_BrakeSubsystem</zorder>
  <zorder>BTN_AuxiliarySubsystem</zorder>
  <zorder>BTN_BatterySubsystem</zorder>
  <zorder>BTN_BogieLampSubsystem</zorder>
  <zorder>BTN_HMIInitSet</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
